### PR TITLE
Development

### PR DESCRIPTION
**Pull request by dianavermilya**

### CODE REVIEW

**Comment by anneCarlson** [comment]
38 files changed. Sheesh.

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <name>indicoio-java</name>
   <groupId>io.indico</groupId>
   <artifactId>io.indico</artifactId>
-  <version>3.4.0</version>
+  <version>3.6.0</version>
```
Comment (anneCarlson) [inline pom.xml]: Why isn't this a full version number tick?

**Comment by anneCarlson** [comment]
See my one, comment, but otherwise looks good!

File: pom.xml (hunk 1 above)
Comment (dianavermilya) [inline pom.xml]: this is a big tick.  The bigger ones are for breaking changes (says madison and chris)

Comment (anneCarlson) [inline pom.xml]: oh, there are 3 components to the number. Doy! Looks great! Go ahead and merge when you are ready!

**Pull request merged**